x/signal/keeper.go

```diff
@@ -109,7 +104,7 @@ func (k *Keeper) TryUpgrade(ctx context.Context, _ *types.MsgTryUpgrade) (*types
 		}
 		upgrade := types.Upgrade{
 			AppVersion:    version,
-			UpgradeHeight: sdkCtx.BlockHeader().Height + k.upgradeHeightDelayBlocks,
+			UpgradeHeight: sdkCtx.BlockHeader().Height + appconsts.UpgradeHeightDelay(version),
```

Comment:
Now we can't override it :(

Reply:
I think it does in that function afaiu, but I did not test

https://github.com/celestiaorg/celestia-app/blob/fbbd9e613628225edae690d91a2525ba8f50c15c/pkg/appconsts/versioned_consts.go#L83-L89

Reply:
Ah, yeah I think you might be right (I had skipped over that part)

We could probably move the string parsing to an `init` function